packages/fakta-fordel-beregningsgrunnlag/src/components/tilkommetAktivitet/TilkommetInntektsforholdField.tsx

```diff
@@ -0,0 +1,126 @@
+import { formHooks, InputField, RadioGroupPanel } from '@navikt/ft-form-hooks';
+import { maxValueFormatted, required } from '@navikt/ft-form-validators';
+import { AktivitetStatus } from '@navikt/ft-kodeverk';
+import { Inntektsforhold } from '@navikt/ft-types/src/beregningsgrunnlagFordelingTsType';
+import { VerticalSpacer } from '@navikt/ft-ui-komponenter';
+import { parseCurrencyInput } from '@navikt/ft-utils';
+import React from 'react';
+import { ArbeidsgiverOpplysningerPerId } from '@navikt/ft-types';
+import { TilkommetAktivitetFormValues } from '../../types/FordelBeregningsgrunnlagPanelValues';
+import styles from './tilkommetAktivitet.less';
+
+interface TilkommetInntektsforholdField {
+  formName: string;
+  index: number;
+  readOnly: boolean;
+  inntektsforholdTilVurdering: Inntektsforhold;
+  arbeidsgiverOpplysningerPerId: ArbeidsgiverOpplysningerPerId;
+}
+
+export const getInntektsforholdIdentifikator = (inntektsforhold: Inntektsforhold | undefined): string => {
+  if (!inntektsforhold) {
+    return '';
+  }
+  let result = inntektsforhold.aktivitetStatus;
+  if (inntektsforhold.arbeidsgiverId) {
+    result += inntektsforhold.arbeidsgiverId;
+  }
+  if (inntektsforhold.arbeidsforholdId) {
+    result += inntektsforhold.arbeidsforholdId;
+  }
+  return result;
+};
+
+const TilkommetInntektsforholdField = ({
+  formName,
+  index,
+  readOnly,
+  inntektsforholdTilVurdering,
+  arbeidsgiverOpplysningerPerId,
+}: TilkommetInntektsforholdField) => {
+  const formMethods = formHooks.useFormContext<TilkommetAktivitetFormValues>();
+
+  const getAktivitetNavn = (inntektsforhold: Inntektsforhold) => {
```

Comment:
Er det nødvendig med to metoder som gjør nesten det samme? Dette ligner jo veldig på det som ligger i TilkommetAktivitetUtils

Reply:
Det var vel får å ikkje vise orgnr for å ha ein meir komprimert aksjonspunkttekst.